cmd/ipfs_routingd/.go-ipfs/config

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
is this config supposed to be here? 

With the privkey here, anyone can pretend to be this client. (not that it matters atm.)